.coveragerc

```diff
@@ -246,6 +246,7 @@ omit =
     homeassistant/components/media_player/squeezebox.py
     homeassistant/components/media_player/vlc.py
     homeassistant/components/media_player/yamaha.py
+    homeassistant/components/media_player/mqtt.py
```

Comment:
All MQTT platforms require 100% test coverage. Please remove this line and add tests.